lib/main.dart

```diff
@@ -36,72 +36,93 @@ class _RandomWordsState extends State<RandomWords> {
         ],
       ),
       body: _buildSuggestions(),
+      floatingActionButton: FloatingActionButton(
+        onPressed: () {
+          setState(() {
+            _suggestions.addAll(generateWordPairs().take(5));
+          }); // refresh page
+        },
+        child: const Icon(Icons.plus_one),
+        backgroundColor: Colors.green,
+      ),
     );
   }
 
   Widget _buildSuggestions() {
     return ListView.builder(
       padding: const EdgeInsets.all(16.0),
       itemBuilder: (context, item) {
-        if (item.isOdd) return const Divider();
-        final index = item ~/ 2;
-        if (index >= _suggestions.length) {
-          _suggestions.addAll(generateWordPairs().take(10));
-        }
-        return _buildRow(_suggestions[index]);
+        return (item < _suggestions.length)
+            ? _buildRow(_suggestions[item])
+            : Container(color: Colors.white);
       },
     );
   }
 
   Widget _buildRow(WordPair pair) {
     final alreadySaved = _saved.contains(pair);
-    return ListTile(
-      title: Text(
-        pair.asPascalCase,
-        style: _biggerFont,
-      ),
-      trailing: Icon(
-        alreadySaved ? Icons.favorite : Icons.favorite_border,
-        color: alreadySaved ? Colors.red : null,
-      ),
-      onTap: () {
-        setState(() {
-          if (alreadySaved) {
-            _saved.remove(pair);
-          } else {
-            _saved.add(pair);
-          }
-        });
-      },
-    );
-  }
-  void _pushSaved() {
-    Navigator.of(context).push(
-      MaterialPageRoute<void>(
-        builder: (BuildContext context) {
-          final tiles = _saved.map(
-              (WordPair pair) {
-                return ListTile(
-                  title: Text(
-                    pair.asPascalCase,
-                    style: _biggerFont,
-                  ),
-                );
-              }
-          );
-          final divided = tiles.isNotEmpty
-              ? ListTile.divideTiles(context: context, tiles: tiles).toList()
-              : <Widget>[];
-          return Scaffold(
-            appBar: AppBar(
-              title: Text('Saved Suggestions'),
+    return (alreadySaved)
+        ? Container(color: Colors.white)
+        : ListTile(
+            title: Text(
+              pair.asPascalCase,
+              style: _biggerFont,
```

Comment:
I think, in this case, you don`t need to use a special variable for fontSize

Reply:
if you left a comment with a fix, than probably it's not that *perfect*. so be careful before leaving comments like that in the future